case_studies/coadds/align_single_exposures.py

```diff
@@ -0,0 +1,39 @@
+# galsim_decoder changes for coadds
+import numpy as np
+import matplotlib.pyplot as plt 
+from pathlib import Path
+import galsim
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+from bliss.models.decoder import get_mgrid
+
+def align_single_exposures(img0, img1, slen, pixel_scale, dithers):
```

Comment:
It would be great if you could add documentation to this function, specifying what each variable is and what are the dimensions of the input tensors